pkg/operator/target_config_reconciler_test.go

```diff
@@ -1530,9 +1741,15 @@ func TestSync(t *testing.T) {
 							},
 						},
 					},
+					&corev1.Namespace{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:   operatorclient.OperatorNamespace,
+							Labels: map[string]string{operatorclient.OpenshiftClusterMonitoringLabelKey: operatorclient.OpenshiftClusterMonitoringLabelValue},
```

Comment:
Just for learning: why do we require the operator namespace to be labelled with "monitoring" ?

Reply:
There is a lovely typo in `checkNamepsaceMonitoringLabel()` :-)

Reply:
Done, thanks.